requirements/base.txt

```diff
@@ -175,7 +175,9 @@ django-rest-framework-condition==0.1.1
 django-sendfile2==0.7.0
     # via django-privates
 django-setup-configuration==0.1.0
```

Comment:
I think you can now remove `django-setup-configuration` from base.in

Reply:
You're right! I also removed `djangorestframework-inclusions`